packages/core/src/Utils/ExchangeCurrencyRate/exchange_currency_rate.js

```diff
@@ -0,0 +1,7 @@
+import BinarySocket from '_common/base/socket_base';
+
+export const getExchangeRate = async (from_currency, to_currency) => {
+    const { exchange_rates } = await BinarySocket.exchange_rates(from_currency, to_currency);
```

Comment:
why are we passing a `to_currency` as a second parameter here? it seems like `BinarySocket.exchange_rates` accepts only 1 param which is `from_currency`, right? 🙂 (socket_base.js line 183)